gammapy/data/event_list.py

```diff
@@ -190,29 +181,15 @@ def add_galactic_columns(self):
         self.table['GLON'] = galactic.l.degree
         self.table['GLAT'] = galactic.b.degree
 
-    # TODO: the following properties are also present on the `DataStoreObservation` class.
```

Comment:
@cosimoNigro During the rebase I wasn't sure if I should keep this part. Do you remember if you put it in?